bim2sim/tasks/common/create_elements.py

```diff
@@ -182,9 +186,19 @@ def create_with_validation(self, entities_dict, warn=True, force=False) -> \
             except LookupError:
                 invalid.append(entity)
                 continue
-
-            self.create_layers_and_materials(element)
-            valid += self.layersets_all + self.layers_all + self.materials_all
+            # TODO #676
+            plugin_name = self.playground.project.plugin_cls.name
+            if plugin_name in ['EnergyPlus', 'Comfort', 'Teaser']:
+                if (self.playground.sim_settings.layers_and_materials
+                        is not LOD.low):
+                    raise NotImplementedError(
+                        "layers_and_materials full is currently not supported.")
```

Comment:
```suggestion
                        "Only layers_and_materials using LOD.low is currently supported.")
```

Reply:
We don't have any other enrichment than LOD.low, right? So I would phrase the error according to the If-statement.